src/tests/additionalTests.spec.js

```diff
@@ -0,0 +1,63 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { SORTING_OPTIONS_FUNCTIONS } = require('../pages/Inventory.page');
+
+test.describe('', () => {
+    test('Perform and verify sorting', async ({ loginPage, inventoryPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+
+        const options = Object.keys(SORTING_OPTIONS_FUNCTIONS);
+
+        for (let i = 0; i < options.length; i++) {
```

Comment:
Let's have verification of the sorting for every single sorting option as a separate test. The reason for that is - if the test will fail you won't understand what sorting caused the failure